src/main/java/uk/gov/hmcts/probate/model/ccd/raw/LegalStatementExecutorApplying.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.probate.model.ccd.raw;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class LegalStatementExecutorApplying {
+
+    @JsonProperty(value = "name")
```

Comment:
@JsonProperty annotations are redundant when the value equals to the variable name.

Reply:
agreed!